gcovr/html_generator.py

```diff
@@ -71,13 +71,36 @@ def load_user_template(template):
 
 class CssRenderer():
 
-    low_color = "LightPink"
-    medium_color = "#FFFF55"
-    high_color = "LightGreen"
-    covered_color = "LightGreen"
-    uncovered_color = "LightPink"
-    takenBranch_color = "Green"
-    notTakenBranch_color = "Red"
+    Themes = {
+        'green': {
+            'unknown_color': "LightGray",
+            'low_color': "#FF4040",
+            'medium_color': "#F9FD63",
+            'high_color': "#85E485",
+            'covered_color': "LightGreen",
+            'uncovered_color': "#FF8C8C",
+            'takenBranch_color': "Green",
+            'notTakenBranch_color': "Red"
+        },
+        'blue': {
+            'unknown_color': "LightGray",
+            'low_color': "#FF4040",
+            'medium_color': "#F9FD63",
+            'high_color': "#6675FF",
+            'covered_color': "dodgerblue",
+            'uncovered_color': "#FF8C8C",
+            'takenBranch_color': "Blue",
+            'notTakenBranch_color': "Red"
```

Comment:
So, regarding the colors of the blue theme: I think the chosen blue colors are a bit too dark to be legible when used as background. I'd suggest replacing `dodgerblue` (`#1e90ff`) and instead use `#66b4ff` for high_color and covered_color.

That color is close to the `LightSteelBlue` used as table backgrounds. But if the blue theme uses `lightgray` for tables instead, the accent colors have stronger effect.

Similarly to the blue backgrounds, we could consider lightening up low_color to `#FF6666` for both themes. This should make the legend a bit easier to read.

But all in all, this configurability is a great feature that will make the reports more accessible.

Reply:
Sounds good, I'll give it a try. But if we use in the blue theme the same for high_color and covered_color this should also be done in the green theme.

Reply:
The lighter colors are better because of the contrast. As mentioned, the `nested` test is a good example because all colors are used and also the row highlighting is available.

Reply:
Thanks! What's you opinion on changing LightSteelBlue for lightgray in table backgrounds in the blue theme? Currently that color isn't yet handled by the theme system, but removing blueish background would improve visibility of blue meters.

Reply:
I'll check this. In the blue theme I had problems with the background.

Reply:
I think the LightSteelBlue is better because the background of the meter is whitesmoke. I've added a little margin at the top of the meter and the blue theme to the test `nested2`.